examples/prometheus/prometheus.yaml

```diff
@@ -18,8 +18,19 @@ parameters:
 - description: The location of the prometheus image
   name: IMAGE_PROMETHEUS
   value: openshift/prometheus:v2.0.0-dev
+- description: The location of the alertmanager image
+  name: IMAGE_ALERTMANAGER
+  # TODO: Change to official openshift build
+  value: prom/alertmanager
+- description: The location of alert-buffer image
+  name: IMAGE_ALERT_BUFFER
+  # TODO: change to official openshift build
+  value: ilackarms/message-buffer
 - description: The session secret for the proxy
-  name: SESSION_SECRET
+  name: PROMETEHUS_SESSION_SECRET
```

Comment:
This is a typo.  Also, why are you splitting them if they are deployed together?